src/gui/static/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,89 @@
+import { NgModule } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163600)
```suggestion
import { NgModule } from "@angular/core";
```